tests/unittests/tests-phydat/tests-phydat.c

```diff
@@ -188,46 +189,137 @@ static void test_phydat_fit(void)
     }
 }
 
+static void test_phydat_unit_write(void)
+{
+    char buffer[] = {'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', '\0',};
```

Comment:
```suggestion
    char buffer[] = {'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', '\0'};
```
?

Reply:
I added that `,` to be more inline with the style of array declaration in the rest of the file, e.g lines 129-142:
```C
static const unsigned int dims[] = {
        1,
        2,
        3,
        3,
        3,
        3,
        3,
        3,
        3,
        3,
        3,
        3,
};
```

Overall, the style of my quick changes here is not that beautiful but it's just tests. I would rather spend my time on other topics if I can get away with this 😇

Reply:
Yeah, but that is so that when adding things at the end the git diff is cleaner (otherwise the last element would need the `,` added). This doesn't apply to the style where the array initializers are in a single line.

In either case: The CI doesn't like the trailing `,` right in front of the `}` either, so you have to adapt anyway ;)